docs/glossary.md

```diff
@@ -5,6 +5,29 @@ sidebar_label: Glossary
 slug: /glossary
 ---
 
+#### Option
+A [call](#call) or a [put](#put), an option is a contract that gives the owner the right to buy (in the case of a call) or sell (in the case of a put) a number of the [underlying asset](#underlying-asset) at a predetermined price ([strike price](#strike-price)) on or before a fixed expiration date.
+
+#### Call
+Buyers of call options generally expect the [underlying asset](#underlying-asset) to increase in value, whereas sellers of calls expect a decrease in value or remain the same.
+
+Buying a call gives you the right, but not the obligation, to buy the [contract size](#contract-size) amount of underlying asset at the option's strike price. Price of the call option should increase as the price of the underlying asset increases.
+
+Selling a call option allows you to collect the premium while obligating you to sell a contract size amount of the underlying asset to the owner at the agreed-upon strike price if the owner of the contract chooses to exercise the contract.
```

Comment:
"Selling" should be re-named to "Writing a covered call" because if I buy a call on Serum, then sell it, I'm just trading the contract. Not actually obligating anything.